src/corner_kick/src/containers/Canvas/__stories__/spritesheet.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file creates a story for the SpritesheetManager
+ */
+
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+
+import { StorybookStage } from 'SRC/components/StorybookStage';
+
+import { exampleSpritesheet } from '../__fixtures__/spritesheet';
+import { SpritesheetManager } from '../spritesheetManager';
+
+const stories = storiesOf('Spritesheet Generator', module);
+
+const spritesheetGenerator = new SpritesheetManager(exampleSpritesheet);
```

Comment:
should we test spritesheet generation as well here? ie. remove the `exampleSpritesheet` and use a generated one?

Reply:
We are testing spritesheet generation here. The exampleSpritesheet is the specification we use to generate the spritesheet.

Reply:
Renamed `exampleSpritesheet` to `exampleSpritesheetFormat` to make this more explicit

Reply:
hmm, could we rename it to `expectedSpritesheet`? I think `expected` better brings to mind the idea of a test reference object.

Reply:
Uh, not sure what is expected. We are talking about the spritesheet format here, might be confusing as we are not checking the format itself, rather the generated end-result.

Reply:
Renamed to `expectedSpritesheet`